### ISSUE TITLE
[BUG] editions/server-external-js is broken

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#7346]
### Describe the bug

@Jermolene ... PR https://github.com/Jermolene/TiddlyWiki5/commit/bef11fe6a25fb849dee40c4aa4337d6a30daf0b4 breaks 5.2.6 prerelease

To replicate: 

```
tiddlywiki ./myNewWiki --init server-external-js
tiddlywiki ./myNewWiki --build listen
```

![image](https://user-images.githubusercontent.com/374655/223953332-230617f1-a19d-45dd-b20b-836cd5ee8dc0.png)

That's how the index.html looks like.

```
<!--~~ Load external JavaScripts ~~-->
<script src="" onerror="alert('Error: Cannot load ');"></script>           <-- src="" .. 
</body>
</html>
```

That's how the template looks like `$:/core/templates/tiddlywiki5-external-js.html`

```
<!--~~ Load external JavaScripts ~~-->
<script src="`{{{ [<coreURL>] }}}`" onerror="alert('Error: Cannot load `{{{ [<coreURL>] }}}`');"></script>
</body>
</html>`
</$set>
```

--------

I'm experimenting atm. .. More info will follow

**Comment by cdruan** [comment]
This issue is fixed by commit 007b759 and can be closed.

**Comment by Jermolene** [comment]
Thanks @cdruan
